From 0b608993d44f2f4a42a3f821991be135995baa88 Mon Sep 17 00:00:00 2001
Message-Id: <0b608993d44f2f4a42a3f821991be135995baa88.1335289128.git.minovotn@redhat.com>
In-Reply-To: <fb97edc658cdc5a2fc0baac7178417d48778534c.1335289128.git.minovotn@redhat.com>
References: <fb97edc658cdc5a2fc0baac7178417d48778534c.1335289128.git.minovotn@redhat.com>
From: Uri Lublin <uril@redhat.com>
Date: Tue, 24 Apr 2012 12:23:16 +0200
Subject: [PATCH 3/3] qxl: PC_RHEL6_1_COMPAT: make qxl default revision value
 2 for -M 6.1.0

RH-Author: Uri Lublin <uril@redhat.com>
Message-id: <1335270196-11253-1-git-send-email-uril@redhat.com>
Patchwork-id: 39421
O-Subject: [RHEL-6.3 qemu-kvm PATCH] qxl: PC_RHEL6_1_COMPAT: make qxl default revision value 2 for -M 6.1.0
Bugzilla: 698936
RH-Acked-by: Alon Levy <alevy@redhat.com>
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>

The default revision value of qxl devices has changed from 2 to 3 by
commit 6fc33d6d0d2370935243a04b00abcd15d0aa8658.

This patch keeps the value 2 for default qxl revision for -M 6.1.0

rhbz#698936
---
 hw/pc.c |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hw/pc.c |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/hw/pc.c b/hw/pc.c
index 60e289b..4371b38 100644
--- a/hw/pc.c
+++ b/hw/pc.c
@@ -1593,6 +1593,14 @@ static void rhel_common_init(const char *type1_version,
             .property = "event_idx",\
             .value    = "off",\
         },{\
+            .driver   = "qxl-vga",\
+            .property = "revision",\
+            .value    = stringify(2),\
+        },{\
+            .driver   = "qxl",\
+            .property = "revision",\
+            .value    = stringify(2),\
+        },{\
             .driver   = "virtio-balloon",\
             .property = "event_idx",\
             .value    = "off",\
-- 
1.7.7.6

